packages/types/src/extension-types.ts

```diff
@@ -81,6 +81,7 @@ export enum ID {
   PAYMENT_NETWORK_ANY_DECLARATIVE = 'pn-any-declarative',
   PAYMENT_NETWORK_ANY_TO_ERC20_PROXY = 'pn-any-to-erc20-proxy',
   PAYMENT_NETWORK_ANY_TO_ETH_PROXY = 'pn-any-to-eth-proxy',
+  PAYMENT_NETWORK_ERC20_NFT_CONTRACT = 'pn-erc20-nft-contract',
```

Comment:
We've had some mixed naming conventions in the past but going forward we're trying to avoid using "_CONTRACT"  at the end of identifiers because it doesn't add any information.

Reply:
will change!